charts/geoweb-opmet-backend/values.yaml

```diff
@@ -49,6 +78,20 @@ opmet:
       limits:
         memory: "100Mi"
         cpu: "1"
+#    livenessProbe:
+#      httpGet:
+#        path: /healthcheck
+#        port: 8090
+#      initialDelaySeconds: 5
+#      periodSeconds: 5
+#      failureThreshold: 5
+#    readinessProbe:
+#      httpGet:
+#        path: /healthcheck?readiness=true
+#        port: 8090
+#      initialDelaySeconds: 5
+#      periodSeconds: 5
+#      failureThreshold: 1
```

Comment:
Why is this commented out?

Reply:
Publisher container doesn't have any healthcheck endpoint available as of now, so can't implement the liveness/readinessProbes yet, but added the support for it so default probes can be easily enabled afterwards by removing the comments.

Reply:
There's user story for creating the missing endpoints: https://gitlab.com/opengeoweb/geoweb-assets/-/issues/2852